stix_shifter_modules/trustar/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,70 @@
+{
+  "x-trustar-query": {
+    "fields": {
+      "x_searchTerm": ["searchTerm"],
+      "x_type": ["entityTypes"],
+      "x_report_details": ["reportId"],
+      "x_report_ids": ["reportIds"],
+      "x_enclaveIds" : ["enclaveIds"],
+      "x_tags": ["tags"],
+      "x_excludeTags": ["x_excludeTags"]
+    }
+  },
+  "x-bitcoin": {
```

Comment:
follow the same naming conventions as `x-trustar-query` for all custom objects such as `x-trustar-bitcoin`

Reply:
also you need to follow the same naming conventions in to_stix_map.

Reply:
@mdazam1942
Implemented as suggested review it and merge it.